spec/validation/loader_spec.rb

```diff
@@ -0,0 +1,80 @@
+# frozen_string_literal: true
+
+#==============================================================================
+# Copyright (C) 2017 Stephen F. Norledge and Alces Software Ltd.
+#
+# This file/package is part of Alces Metalware.
+#
+# Alces Metalware is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License
+# as published by the Free Software Foundation, either version 3 of
+# the License, or (at your option) any later version.
+#
+# Alces Metalware is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this package.  If not, see <http://www.gnu.org/licenses/>.
+#
+# For more information on the Alces Metalware, please visit:
+# https://github.com/alces-software/metalware
+#==============================================================================
+
+require 'validation/loader'
+
+RSpec.describe Metalware::Validation::Loader do
+  describe '#configure_data' do
+    let! :config do
```

Comment:
Not that this is required for this Spec, but you can also use `include AlcesUtils` to set up the `config`. It will also mocks `Alces.new` to always return the same `alces` object which makes testing it easier.

As this spec doesn't need to use `alces`, it is six of one half dozen of the other.

Reply:
Thanks for the info, I'll use `AlcesUtils` if I need to in future tests :slightly_smiling_face:.